tests/Api/ProductMediaFile/DownloadProductMediaFileApiIntegration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Akeneo\Pim\tests\Api\ProductMediaFile;
+
+use Akeneo\Pim\tests\Api\ApiTestCase;
+use Psr\Http\Message\StreamInterface;
+
+class DownloadProductMediaFileApiIntegration extends ApiTestCase
+{
+    public function testDownload()
+    {
+        $api = $this->createClient()->getProductMediaFileApi();
+        $expectedMediaFile = realpath(__DIR__ . '/../../fixtures/akeneo.png');
+
+        $api->create($expectedMediaFile, [
+            'identifier' => 'medium_boot',
+            'attribute'  => 'side_view',
+            'scope'      => null,
+            'locale'     => null,
+        ]);
+
+        $mediaFiles = $api->listPerPage()->getItems();
+        $mediaFile = $api->download($mediaFiles[4]['code']);
```

Comment:
add  `TODO: use code returned by creation when API-263 done` please. Because the harcoded index is not clean.